modules/social_features/social_group/social_group.module

```diff
@@ -725,7 +725,7 @@ function social_group_form_alter(&$form, FormStateInterface $form_state, $form_i
         }
       }
 
-      $form['#fieldgroups']['group_settings']->children[] = 'group_type';
+      $form['#fieldgroups']['group_content']->children[] = 'group_type';
```

Comment:
This will change the region where changing the Group type should appear. 

**Before this change:**

![image](https://user-images.githubusercontent.com/8435994/98817292-32890900-242a-11eb-9740-d1250a1c71a6.png)


**After this change:**

![image](https://user-images.githubusercontent.com/8435994/98817379-4e8caa80-242a-11eb-9bcd-e4e1da22f024.png)

Reply:
Good find @navneet0693 - didn't notice that!
@diakivvasyl could you take a look to that?